mmv1/products/compute/Subnetwork.yaml

```diff
@@ -210,17 +210,23 @@ properties:
     immutable: true
     required: true
     custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.erb'
-  - !ruby/object:Api::Type::String
+  - !ruby/object:Api::Type::Enum
     name: 'purpose'
     immutable: true
+    values:
+      - :PRIVATE
+      - :REGIONAL_MANAGED_PROXY
+      - :GLOBAL_MANAGED_PROXY
+      - :PRIVATE_SERVICE_CONNECT
+      - :PRIVATE_NAT
```

Comment:
```suggestion
```

Reply:
Hi, thanks for your review. I removed the commit containing the var change and only kept the documentation updates.